src/components/autosuggest/_macro.spec.js

```diff
@@ -190,107 +237,70 @@ describe('macro: autosuggest', () => {
-            expect(inputSpy.occurrences[0]).toHaveProperty('legend', 'this is a legend');
-            expect(inputSpy.occurrences[0]).toHaveProperty('legendClasses', 'legend-extra-class');
-            expect(inputSpy.occurrences[0]).toHaveProperty('value', 'abc');
-            expect(inputSpy.occurrences[0]).toHaveProperty('attributes.a', 42);
-            expect(inputSpy.occurrences[0]).toHaveProperty('error.id', 'error-id');
-            expect(inputSpy.occurrences[0]).toHaveProperty('error.text', 'An error occurred.');
-            expect(inputSpy.occurrences[0]).toHaveProperty('mutuallyExclusive', null);
-            expect(inputSpy.occurrences[0]).toHaveProperty('accessiblePlaceholder', true);
-            expect(inputSpy.occurrences[0]).toHaveProperty('name', 'test-params');
-            expect(typeof inputSpy.occurrences[0].autosuggestResults).toBe('string');
-            expect(inputSpy.occurrences[0]).toHaveProperty('minLength', 1);
-            expect(inputSpy.occurrences[0]).toHaveProperty('maxLength', 10);
-            expect(inputSpy.occurrences[0]).toHaveProperty('prefix.title', 'Great British Pounds');
-            expect(inputSpy.occurrences[0]).toHaveProperty('prefix.text', '£');
-            expect(inputSpy.occurrences[0]).toHaveProperty('prefix.id', 'gbp-prefix');
-            expect(inputSpy.occurrences[0]).toHaveProperty('suffix.title', 'Percentage of total');
-            expect(inputSpy.occurrences[0]).toHaveProperty('suffix.text', '%');
-            expect(inputSpy.occurrences[0]).toHaveProperty('suffix.id', 'percentage-suffix');
-            expect(inputSpy.occurrences[0]).toHaveProperty('fieldId', 'field-id-test');
-            expect(inputSpy.occurrences[0]).toHaveProperty('fieldClasses', 'field-class-test');
-            expect(inputSpy.occurrences[0]).toHaveProperty('dontWrap', true);
-            expect(inputSpy.occurrences[0]).toHaveProperty('charCheckLimit.limit', 200);
-            expect(inputSpy.occurrences[0]).toHaveProperty('charCheckLimit.charCountSingular', 'You have {x} character remaining');
-            expect(inputSpy.occurrences[0]).toHaveProperty('charCheckLimit.charCountPlural', 'You have {x} characters remaining');
-            expect(inputSpy.occurrences[0]).toHaveProperty('charCheckLimit.charCountOverLimitSingular', '{x} character too many');
-            expect(inputSpy.occurrences[0]).toHaveProperty('charCheckLimit.charCountOverLimitPlural', '{x} characters too many');
-            expect(inputSpy.occurrences[0]).toHaveProperty('searchButton.text', 'Search');
-            expect(inputSpy.occurrences[0]).toHaveProperty('postTextboxLinkText', 'Post textbox link text');
-            expect(inputSpy.occurrences[0]).toHaveProperty('postTextboxLinkUrl', 'https://www.ons.gov.uk');
-            expect(inputSpy.occurrences[0]).toHaveProperty('listeners.click', "function() { console.log('click'); }");
+            test('THEN: it renders the input component with the expected parameters', () => {
```

Comment:
Like the address-input test this should probably be "THEN: the provided attributes are passed through to the input component" because that is what it is testing